deploy/crds/keycloak.org_keycloakrealms_crd.yaml

```diff
@@ -1141,6 +1138,9 @@ spec:
                         type: string
                     type: object
                   type: array
+                userManagedAccessAllowed:
+                  description: User Managed Access Allowed
+                  type: boolean
```

Comment:
This is an implicit fix (made by `make code/gen`). In #346 the  change to the CRD was apparently made manually and at a wrong place (in clients section, not realm).

Reply:
It's fine.